install/overlays/ibmcloud/kata_direct_volumes_mount.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: cloud-api-adaptor-daemonset
+  namespace: confidential-containers-system
+  labels:
+    app: cloud-api-adaptor
+spec:
+  template:
+    spec:
+      containers:
+      - name: cloud-api-adaptor-con
+        volumeMounts:
+        - mountPath: /run/kata-containers/shared/direct-volumes
+          name: kata-direct-volumes-dir
+      volumes:
+      - name: kata-direct-volumes-dir
+        hostPath:
+          path: /run/kata-containers/shared/direct-volumes
+          type: DirectoryOrCreate
+
+# to apply this uncomment the patchesStrategicMerge of this file in kustomization.yaml
```

Comment:
In kustomization.yaml the `- kata_direct_volumes_mount.yaml` already uncommented.
So is this line still need?

Reply:
maybe comment it in kustomization.yaml is better.